Core/Graphics/InternalInclude/Babylon/Graphics/DeviceContext.h

```diff
@@ -121,6 +121,9 @@ namespace Babylon::Graphics
         TextureInfo GetTextureInfo(bgfx::TextureHandle handle);
         static bx::AllocatorI& GetDefaultAllocator() { return m_allocator; }
 
+        // call following method when a submit call is associated with the current acquired viewId
+        void SetViewAsUsed();
```

Comment:
I guess we have to expose this new function because `FrameBuffer` has to tell `DeviceContext` this needs to be done when a `Submit` happens. Is there any other case (other than `Submit`) where we would need to do this, or is this really tied to `FrameBuffer::Submit`?

If we need to expose this as a new function, would `InvalidateView` be a better name?

Reply:
Yes, Framebuffer tells device context the view contains draw call. I've added a call for `Clear` . `InvalidateView` works for me!